etl/src/main/java/org/pih/hivmigration/etl/sql/EncounterMigrator.groovy

```diff
@@ -177,14 +181,12 @@ class EncounterMigrator extends SqlMigrator {
         ''')
 
         executeMysql("Log warnings for encounters with null encounter dates", '''
-            INSERT INTO hivmigration_data_warnings (patient_id, encounter_id, field_name, note)
+            INSERT INTO hivmigration_data_warnings (openmrs_patient_id, openmrs_encounter_id, field_name, warning_type, warning_details)
             SELECT p.person_id,
                    e.encounter_id,
                    'encounter_date',
-                   CONCAT('Encounter date is null. `date_created` used as encounter date. ',
-                          'Source encounter_id ', e.source_encounter_id,
-                          '. Source patient_id ', e.source_patient_id,
-                          '. Source encounter_type ', e.source_encounter_type, '.')
+                   'Encounter date is null. `date_created` used as encounter date.',
+                   CONCAT('Source encounter_type :', e.source_encounter_type)
```

Comment:
Removed source_encounter_id and source_patient_id as they now have their own columns.